.github/workflows/review.yml

```diff
@@ -130,6 +130,7 @@ jobs:
         uses: actions/checkout@v4
         with:
           repository: NixOS/nixpkgs
+          path: nixpkgs
```

Comment:
The alternative would be to put the other (sparse) checkout in a different `path`. But this is fine too.